docs/.vitepress/theme/plane/css/base.css

```
@@ -0,0 +1,123 @@
+/* ================================================
+   PLANE DOCS THEME — base typography, code, misc doc content
+   ================================================ */
+
+/* --- Font rendering --- */
+body,
+.docs-layout {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)